start-kafka.sh

```diff
@@ -39,7 +39,25 @@ done
 $KAFKA_HOME/bin/kafka-server-start.sh $KAFKA_HOME/config/server.properties &
 KAFKA_SERVER_PID=$!
 
-while netstat -lnt | awk '$4 ~ /:9092$/ {exit 1}'; do sleep 1; done
+
+if [[ -z "$KAFKA_START_WAIT_TIMEOUT" ]]; then
```

Comment:
Variables with the KAFKA_ prefix should be reserved for configuration that goes into server.properties. Once we've merged this PR I will update "KAFKA_CREATE_TOPICS" and the documentation